src/rulesets/oas3/index.json

```
@@ -83,7 +83,20 @@
       "message": "{{error}}",
       "recommended": true,
       "type": "validation",
-      "given": "$..[?(@.example)]",
+      "given": "$..[?(@.example && !@.type)]",
```

I was about to suggest that you could try narrowing it down by `schema` but then I realized it won't work out of the box. There would still be a corner case where someone has an example but didn't provide a schema.

On a second thought I will suggest it :)

```suggestion
      "given": "$..[?(@.example && !@.schema)]",
```

If I understand this query correctly it will match all 'nodes' that have `example` but doesn't have `schema` next to it. This could mean two things:
- either the `example` you found is **inside** the `schema`
- or the `example` should be next to `schema` but that `schema` was not provided!

If it was not provided you will get `undefined` but that's fine because you should be able to check that in the actual rule implementation.